src/measure_theory/integral/interval_integral.lean

```diff
@@ -1425,6 +1425,28 @@ end mono
 
 end
 
+section has_sum
+variables {μ : measure ℝ} {f : ℝ → E}
+
+lemma _root_.measure_theory.integrable.has_sum_interval_integral (hfi : integrable f μ) (y : ℝ) :
+  has_sum (λ (n : ℤ), ∫ x in (y + n)..(y + n + 1), f x ∂μ) (∫ x, f x ∂μ) :=
+begin
+  simp_rw integral_of_le (le_add_of_nonneg_right zero_le_one),
+  rw [←integral_univ, ←Union_Ioc_add_int_cast y],
+  exact has_sum_integral_Union (λ i, measurable_set_Ioc) (pairwise_disjoint_Ioc_add_int_cast y)
+    hfi.integrable_on,
+end
+
+lemma _root_.measure_theory.integrable.has_sum_interval_integral_comp_add_int
+  (hfi : integrable f) :
```

Comment:
Is the omission of a `μ` argument here deliberate?

Reply:
The statement is clearly false unless `μ` is invariant under translation by integers. One could state it for a generic `μ` satisfying `[μ.is_add_right_invariant]`, but since the only translation-invariant locally-finite Borel measure on `ℝ` is the canonical volume measure, the added generality would be illusory. Which is, I expect, precisely why docs#interval_integral.integral_comp_add_right (the lemma used in the proof) is only stated for `volume` as well.

Reply:
Sorry for the distraction, you're obviously right here.